integrations/terraform/envs/portal/locals.tf

```diff
@@ -26,9 +20,15 @@ locals {
   data_outputs_raw = yamldecode(file("${path.root}/config/data_glossary/data_outputs.yaml"))
   data_outputs = {
     for k, v in local.data_outputs_raw : k => {
-      s3    = try(v["s3"], [])
-      glue  = try(v["glue"], [])
-      owner = try(v["owner"], [])
+      s3 = try([{
+        bucket_identifier = v["S3"]["bucket"]
```

Comment:
Previous remarks will impact this as well of course

Reply:
Fixed with latest commit. Platform and service names will always be exported as lowercase, also reflected this in all example files.